dts/bindings/display/fsl,imx6sx-lcdif.yaml

```diff
@@ -1,11 +1,11 @@
-# Copyright (c) 2019, NXP
+# Copyright (c) 2019-2022, NXP
 # SPDX-License-Identifier: Apache-2.0
 
 description: NXP i.MX eLCDIF (Enhanced LCD Interface) controller
 
 compatible: "fsl,imx6sx-lcdif"
```

Comment:
Should we change this compatible string to reflect the fact that the `lcdif` IP block is present on devices besides the iMX6 line?